go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/anoideaopen/acl
 go 1.18
 
 require (
-	github.com/anoideaopen/foundation v0.0.0-20240411055802-53b67be060ea
+	github.com/anoideaopen/foundation v0.0.2-0.20240603112837-d0830e597d52
```

Comment:
I wonder why there's a zero with a period after the hyphen.

Reply:
My IDE calculates it by hash of a commit. Try it locally